cores/esp8266/TypeConversion.h

```diff
@@ -0,0 +1,78 @@
+/*
+   TypeConversion functionality
+   Copyright (C) 2019 Anders Löfgren
+
+   License (MIT license):
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to deal
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+*/
+
+#ifndef __ESP8266_TYPECONVERSION_H__
+#define __ESP8266_TYPECONVERSION_H__
+
+#include <Arduino.h>
+
+namespace esp8266 {
+  namespace TypeConversion {
+    extern const char base36Chars[36];
```

Comment:
Maybe these could be static and local to the TC.cpp file?  Not seeing the use in general code.

Reply:
The base36 arrays are used in a few other type conversion functions which based on conversations with @devyte may end up in separate classes. I thought it could be a good idea to put the arrays in TypeConversion for use in all such conversion functions, but I'm open to other ideas.

Reply:
What is usually done in such cases is to move such arrays into another cpp, and export them in a .h which then gets included wherever.
I think we can clean that up later, this is all in experimental so we can make changes without having to explain or jump through hoops.